scripts/test-licenses.js

```diff
@@ -41,6 +41,7 @@ const acceptedSpdxLicenses = [
   'BSD-4-Clause',
   'CC-BY-3.0',
   'CC-BY-4.0',
+  'CC0-1.0',
```

Comment:
Unfortunately, we cannot allow `CC0` licenses in any Google project. It is not a valid license. I'm looking at why your PR would not work without this whitelisting.

Reply:
CC0 is a well-defined dedication of the work to the Public Domain.  I would suspect that this would be more accepted than a generically worded Public Domain dedication.  One of the parts of postcss-cssnext is a Public Domain component.  I would argue it to be similar to "Beerware" or the "Unlicense", although a lot more complete.

The offending packages are:
- postcss-nesting@4.1.0
- postcss-pseudo-class-any-link@4.0.0

Here's a link to one license text: https://github.com/jonathantneal/postcss-nesting/blob/master/LICENSE.md